src/Orleans/Orleans.csproj

```diff
@@ -71,27 +71,18 @@
     <WarningsAsErrors>4014</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Data.Edm">
```

Comment:
@sergeybykov - Sure thing, i will squash. I just did things are seperate incremental commits to make sure i didn't break anything along the way :)